common/lib/dependabot/notifications.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "active_support/notifications"
+
+module Dependabot
+  def self.instrument(name, payload = {})
```

Comment:
I didn't want to spill `ActiveSupport::Notifications` all over the place, but not sure if this adds much value looking at it again, if we ever decide to replace this with some other system we most likely will need to change the interface anyway, and it just hides information from a reader of the code.

Reply:
Seems ok to wrap it up, feel like you always end up wanting to wrap stuff for different reasons in the end